packages/jest-worker/src/index.ts

```diff
@@ -72,18 +74,23 @@ export default class JestWorker {
   private _farm: Farm;
   private _options: FarmOptions;
   private _workerPool: WorkerPoolInterface;
+  private _inspector: Session | undefined;
```

Comment:
```suggestion
  private _inspectorSession: Session | undefined;
```

easier to grok when we have the `inspector` import